hepsiburada.js.org

berkaltiok opened the pull request:
- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

(The aim of this project is to develop an e-commerce theme. I am currently coding the header. I will soon switch to content.)

joeleeofficial commented:
i don't think its related to js.

berkaltiok commented:
> i don't think its related to js.
VueJS is used in the project. My goal is to create an e-commerce pattern for vue. Components, drop-down menus etc.

MattIPv4 reviewed (changes requested):
Hi there!

Before we can accept your domain request, you need to create a file named `CNAME` in the root of your github pages site (_this may not be the root of your repository_). 📝
This file should contain a single line with the full js.org domain you are requesting, in this instance `hepsiburada.js.org`.

berkaltiok commented:
I did the CNAME update

berkaltiok reviewed (commented):
.

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. But your page has path issues now. Please fix them to give me a a chance to have a look at your content.

joeleeofficial commented:
> 
> 
> > i don't think its related to js.
> 
> VueJS is used in the project. My goal is to create an e-commerce pattern for vue. Components, drop-down menus etc.

i see

berkaltiok commented:
> I've already added your requested subdomain to JS.ORGs zonefile. But your page has path issues now. Please fix them to give me a a chance to have a look at your content.

Fixed issue, now it works https://hepsiburada.js.org/
